src/views/active.js

```diff
@@ -90,21 +92,23 @@ export const Active = () => {
         {
             name: 'Creation Time',
             selector: 'creation_time',
-            sortable: true
+            sortable: true,
+            grow: 2
         },
         {
             name: 'CPU',
-            selector: 'cpus',
+            selector: (row) => { return row.cpus / 1000 + ' Core' + (row.cpus / 1000 > '1' ? 's' : '') },
             sortable: true
         },
         {
             name: 'GPU',
-            selector: 'gpus',
+            selector: (row) => { return row.gpus / 1000 + ' Core' + (row.gpus / 1000 > '1' ? 's' : '') },
             sortable: true
         },
         {
             name: 'Memory',
-            selector: 'memory',
+            // Note: Memory in the response does not contain unit, but it matches the one when we post them (MB by default).
+            selector: (row) => { return formatMemory(row.memory + 'M') },
```

Comment:
I think Steve will ask for this to be in GB to keep the number more human friendly and based on 617. We should probably also update Tycho to include the memory type on the instance, or guarantee to serve in MB, then the frontend can let the format function operate to get the most human friendly unit version.

Reply:
I read this wrong, and testing showed me that. Sorry. LGTM.